Makefile

```diff
@@ -329,11 +339,14 @@ goimports:
 	go install golang.org/x/tools/cmd/goimports@latest
 	goimports -w -local="sigs.k8s.io/cluster-api-provider-kubevirt"  $(shell find . -type f -name '*.go' ! -path "*/vendor/*" ! -path "./_kubevirtci/*" ! -path "*zz_generated*" )
 
+GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
+golangci-lint: ## Download golangci-lint locally if necessary.
+	$(call go-install-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/cmd/golangci-lint@v1.44.0)
```

Comment:
please change to the latest version, v1.54.1

Reply:
It's not possible unless we upgrade to Go 1.20

```
$: go version
go version go1.18.8 linux/amd64

$: make linter

Installing github.com/golangci/golangci-lint/cmd/golangci-lint@v1.54.1
# honnef.co/go/tools/unused
../../../.gvm/pkgsets/go1.18.8/global/pkg/mod/honnef.co/go/tools@v0.4.3/unused/unused.go:419:14: obj.Origin undefined (type *types.Var has no field or method Origin)
../../../.gvm/pkgsets/go1.18.8/global/pkg/mod/honnef.co/go/tools@v0.4.3/unused/unused.go:421:14: obj.Origin undefined (type *types.Func has no field or method Origin)
note: module requires Go 1.19
# honnef.co/go/tools/staticcheck
../../../.gvm/pkgsets/go1.18.8/global/pkg/mod/honnef.co/go/tools@v0.4.3/staticcheck/lint.go:4025:29: righti.Method(i).Origin undefined (type *types.Func has no field or method Origin)
../../../.gvm/pkgsets/go1.18.8/global/pkg/mod/honnef.co/go/tools@v0.4.3/staticcheck/lint.go:4030:36: sel.Obj().(*types.Func).Origin undefined (type *types.Func has no field or method Origin)
note: module requires Go 1.19
make: *** [Makefile:344: golangci-lint] Error 2
```